apps/frontend/src/pages/friends/components/friend/friend.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button, Image } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	user: { fullName: string; imageUrl: string };
```

Comment:
```suggestion
	user: friend-type;
```
should be similar to "course" PR